server/services/datacommons.py

```diff
@@ -263,6 +263,55 @@ def v2node(nodes, prop):
   })
 
 
+def _merge_paged_response(result, paged_response):
+  for dcid in paged_response.get('data', []):
+    # Initialize dcid in data even when no arcs or properties are returned
+    result.setdefault('data', {}).setdefault(dcid, {})
+
+    for prop in paged_response['data'][dcid].get('arcs', {}):
+      result['data'][dcid].setdefault('arcs', {}).setdefault(
+          prop, {}).setdefault('nodes', []).extend(
+              paged_response['data'][dcid]['arcs'][prop]['nodes'])
+
+    if 'properties' in paged_response['data'][dcid]:
+      result['data'][dcid].setdefault('properties', []).extend(
+          paged_response['data'][dcid].get('properties', []))
+
+  result['nextToken'] = paged_response.get('nextToken', '')
+  if not result['nextToken']:
+    del result['nextToken']
+
+
+def v2node_paginated(nodes, prop, max_pages=1):
```

Comment:
since this function is for fetching paginated v2node, maybe default for max_pages should be None so when this function does get called, user has to consciously decide to only fetch a specific number of pages?

Reply:
I see what you mean, that makes sense to me